soc/st/stm32/stm32u0x/soc.c

```diff
@@ -20,6 +20,7 @@
 #define LOG_LEVEL CONFIG_SOC_LOG_LEVEL
 LOG_MODULE_REGISTER(soc);
 
+extern void stm32_power_init(void);
```

Comment:
I would suggestion to move this declaration in local **soc.h**.

Reply:
I did apply the suggestion, enclosing the declaration in a `#ifdef CONFIG_PM`.

Should I revert?